test/SlowTests/Issues/RavenDB_17018.cs

```diff
@@ -0,0 +1,268 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Blittable;
+using MongoDB.Bson;
+using Orders;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Operations.Revisions;
+using Raven.Client.Documents.Session;
+using Raven.Server.Config;
+using Raven.Server.NotificationCenter;
+using Raven.Server.NotificationCenter.Notifications;
+using Sparrow.Json;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_17018 : RavenTestBase
+    {
+        public RavenDB_17018(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task ShouldStoreTotalDocumentSizeInPerformanceHint_ForLoadDocuments()
+        {
+            using (var store = GetDocumentStore(new Options
+            {
+                ModifyDatabaseRecord = record => record.Settings[RavenConfiguration.GetKey(x => x.PerformanceHints.MaxNumberOfResults)] = "1"
+            }))
+            {
+                using (var session = store.OpenSession())
+                {
+                    session.Store(new Company { Name = "HR", Address = new Address { Country = "Japan" } }, "companies/1");
+                    session.Store(new Company { Name = "CF", Address = new Address { Country = "Russia" } }, "companies/2");
+                    session.SaveChanges();
+                }
+
+                using (var session = store.OpenSession())
+                {
+                    var companies = session.Load<Company>(new[] {"companies/1", "companies/2"}).ToList();
+                    Assert.NotEmpty(companies);
+                }
+
+                var database = await GetDatabase(store.Database);
+                database.NotificationCenter.Paging.UpdatePaging(null);
+
+                using (database.NotificationCenter.GetStored(out var actions))
+                {
+                    var action = actions.First();
+                    Assert.True(action.Json.TryGet("Details", out BlittableJsonReaderObject details));
+                    Assert.True(details.TryGet("Actions", out BlittableJsonReaderObject detailsActions));
+                    Assert.True(detailsActions.TryGet("GetDocumentsByIdAsync", out BlittableJsonReaderArray detailsArray));
+                    var index = detailsArray.GetByIndex<BlittableJsonReaderObject>(0);
+                    Assert.NotNull(index);
+                    Assert.True(index.TryGet("TotalDocumentsSizeInBytes", out int size));
+                    Assert.Equal(527, size);
```

Comment:
Better to note hard code this value, it is something that may change when we modify internal details.

That said, where are we actually *showing* this?